src/App.js

```diff
@@ -1,89 +1,43 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
-import { SplitFactory } from '@splitsoftware/splitio';
+import React from 'react';
+import { Route, Link, BrowserRouter } from 'react-router-dom'
```

Comment:
Also minor, but missing `;`. Maybe you can run linting from VS Code without adding more to this project?